Dockerfile

```
@@ -0,0 +1,26 @@
+FROM python:3.9-slim
+
+WORKDIR /app
+RUN pip install --upgrade pip
```

Hadolint (2.12.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)